external/anomaly/ote/configs/ote_anomaly_classification_stfpm.yaml

```diff
@@ -0,0 +1,37 @@
+name: Anomaly Classification STFPM
```

Comment:
I think, instead of naming the template filename we need this naming inside the `template.yaml` file. See [this](https://gitlab.devtools.intel.com/vmc-eip/IMPT/impt/-/blob/develop/src/ote_sdk/ote_sdk/entities/model_template.py#L484) model template parser, where it checks `model_template_id` in the yaml file. If it doesn't exist, it'll check `name` in the yaml file. It then replaces " " with "_". 

In the backend, the padim model is registered as `ote_anomaly_classification_padim`. Therefore, we need this either in `model_template_id` in the yaml file, or replace the name in the yaml file

Reply:
so the template paths could be like:

- external/anomalib/anomaly_classification/configs/padim/template.yaml
- external/anomalib/anomaly_classification/configs/stfpm/template.yaml

in padim/template.yaml,

```
name: ote anomaly classification padim
```